calificativ.js

```diff
@@ -13,4 +13,16 @@
      3: IS
      2: IS
      1: IS
-*/
\ No newline at end of file
+*/
+const grade = function (mark) {
+  if (mark <= 4) {
+    return "IS";
+  } else if (mark <= 6) {
+    return "S";
+  } else if (mark <= 8) {
+    return "B";
+  } else {
+    return "FB";
+  }
+};
+console.log(grade("1"));
```

Comment:
Poți apela direct cu număr. Altfel e OK.

```suggestion
console.log(grade(1));
```